### PR TITLE
Language sort optionslist at power tab

### PR DESCRIPTION
**Pull request by HThuren**
Sort using new common function sortSelect at power tab

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Review by haslinghuis — APPROVED** [review]

**Review by McGiverGim — APPROVED** [review]

**Comment by HThuren** [comment]
Thank you

**Pull request merged**
